apps/gitrekt/lib/gitrekt/git_agent.ex

```diff
@@ -755,7 +755,16 @@ defmodule GitRekt.GitAgent do
       {:ok, name, shorthand, oid} ->
         {:ok, resolve_reference({name, shorthand, :oid, oid})}
       {:error, reason} ->
```

Comment:
What kind of error are you getting here? And how doe it happen?

Reply:
> What kind of error are you getting here? And how doe it happen?

Refer to the third paragraph of this comment: https://github.com/redrabbit/git.limo/pull/87#issuecomment-1299463541 

Reproduction steps:
1. Create a new repository `Test` on git.limo.
2. Create a local repository `Test` with only one branch `main` as default branch.
3. Add git.limo `Test` repository as local repository's remote and `git push -u origin main`.
4. Access `Test` repository at git.limo on web.

eg: https://git.limo/edmondfrank/Test